cmd/action/verify/cluster/hasetup/runner.go

```diff
@@ -34,7 +58,7 @@ func (e *Executor) execute(ctx context.Context) error {
 	{
 		c := client.TenantClusterConfig{
 			ControlPlane: cpClients,
-			Logger:       e.logger,
+			Logger:       r.logger,
 		}
```

Comment:
This is the tenant client exception that I tried to explain earlier. To make this work we also need to get the `flag.go` right in this command package. 

```suggestion
			TenantCluster: r.flag.TenantCluster,
		}
```

Reply:
Yeah, I was kind of confused because it was not really as the others will fix now